state_machine/flight_manager.py

```diff
@@ -37,9 +38,14 @@ class FlightManager:
     def __init__(self) -> None:
         """Initialize a flight manager object."""
 
-    def start_manager(self) -> None:
-        """Test running the state machine in a separate process."""
+    def start_manager(self, simflag: bool) -> None:
+        """Test running the state machine in a separate process.
+        Sets the drone address to the simulation or physical address."""
         drone_obj: Drone = Drone()
+        if simflag is True:
```

Comment:
Same here, needs snake_case
```suggestion
        if sim_flag is True:
```